packages/ui/src/accounts/components/AccountItem/components/LocksDetails.tsx

```diff
@@ -15,19 +16,27 @@ interface LocksDetailsProps {
 
 export const LocksDetails = ({ balance, address }: LocksDetailsProps) => {
   const { election } = useCurrentElection()
+  const { votes } = useMyCastVotes()
```

Comment:
Sorry this is on me but the `useMyCastVotes` hook is currently very confusing. It won't return anything unless you provide the `cycleId`. So I think this hook won't work by itself.

Replacing the `useCurrentElection` by a hook which queries something like this:
```gql
query LatestElection {
	electionRounds (orderBy: [cycleId_DESC], limit: 1) {
    ...ElectionRoundFields
  }
}
```
Could work because you would still know whether the election is finished (for `isActiveCandidate`) and the `cycleId` would be available so you could query the votes only if the election is finished.

But the production is currently in a weird state it won't really work there ATM